core/bin/external_node/src/config/observability.rs

```diff
@@ -30,12 +30,14 @@ pub fn observability_config_from_env() -> anyhow::Result<ObservabilityConfig> {
     } else {
         "plain".to_string()
     };
+    let log_directives = std::env::var("RUST_LOG").ok();
```

Comment:
Does this mean we are okay with leaving observability stuff to be configured based on env vars?

Reply:
I also don't understand entirelly.. Do we have it somewhere written? Or can you explain how env and file initialization go along From this code it looks like we read this `log_directives` _only_ from env

Reply:
This method names: 
`observability_from_env` and it's used in external_node because we rename envs data there.

Reply:
To elaborate:
- One can construct `ObservabilityConfig` and use it with `vlog` in a way that they prefer, be it env variables or files or just hardcoding.
- This method is dedicated to fetch configs from `env` so we get this value from env in a way that we did before (e.g. backward compatibility).
- `vlog` still would fallback to env configuration if nothing wat provided, which makes sense given that `RUST_LOG` is an industry default.